conference/react-js/app/src/ComponentB/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default () => (
```

Comment:
Please use normal and not anonymous function. Because in ReactDevTools in browser you will see node with name ```Undefined``` instead of ```ComponentB```

Reply:
Also, you must choose exporting style: in the end of file or in place of function declaration, and use chosen option everywhere.

Reply:
Function is anonimus yet